hydromt_wflow/wflow_sediment.py

```diff
@@ -580,7 +580,7 @@ def setup_lulcmaps(
                 return
             usle_c = workflows.add_planted_forest_to_landuse(
                 planted_forest,
-                self.grid,  # TODO should have USLE_C in the grid already
+                self.grid,  # TODO should have erosion_usle_c in the grid already
```

Comment:
This might need attention now, ie we need to rename self.grid before passing it to this function. See example in https://github.com/Deltares/hydromt_wflow/blob/10cc12ac1db74e90d68b4f128b1e039520a2cd24/hydromt_wflow/wflow.py#L3779-L3783